internal/builder/packages.go

```diff
@@ -233,7 +240,53 @@ func signZippedPackage(options BuildOptions, zippedPackagePath string) error {
 	return nil
 }
 
+func copyLicenseTextFile(licensePath string) error {
```

Comment:
I guess that you don't check if the license content has changed?

Reply:
For example, the `check` command fails if README.md is outdated due to changed fields.

Reply:
Ok 👍